pg_search/src/index/reader/index.rs

```diff
@@ -695,6 +774,49 @@ impl SearchIndexReader {
             .expect("search should not fail")
     }
 
+    /// Create erased Features for the given OrderByInfo.
+    ///
+    /// To avoid a combinatorial explosion of generated code we only specialize the first orderby
+    /// column: the remainder have their types erased.
+    fn erased_features(
+        &self,
+        orderby_infos: Option<&Vec<OrderByInfo>>,
+    ) -> Vec<(ErasedFeature, SortDirection)> {
+        let remainder = orderby_infos.and_then(|oi| oi.get(1..)).unwrap_or(&[]);
+        remainder
+            .iter()
+            .map(|orderby_info| match orderby_info {
+                OrderByInfo {
+                    feature: OrderByFeature::Field(sort_field),
+                    direction,
+                } => {
+                    let field = self
+                        .schema
+                        .search_field(sort_field)
+                        .expect("sort field should exist in index schema");
+
+                    let feature = match field.field_entry().field_type().value_type() {
+                        tantivy::schema::Type::Str => FieldFeature::string(sort_field).erased(),
+                        tantivy::schema::Type::U64 => FieldFeature::u64(sort_field).erased(),
+                        tantivy::schema::Type::I64 => FieldFeature::i64(sort_field).erased(),
+                        tantivy::schema::Type::F64 => FieldFeature::f64(sort_field).erased(),
+                        tantivy::schema::Type::Bool => FieldFeature::bool(sort_field).erased(),
+                        tantivy::schema::Type::Date => FieldFeature::datetime(sort_field).erased(),
+                        x => {
+                            panic!("Unsupported order-by field type: {x:?}");
```

Comment:
Is this something a user might hit with `SELECT ... ORDER BY inet_field` or are we otherwise making sure we don't pass down types that we _can_ index but _not_ sort?